examples/features/metadata/client/main.go

```diff
@@ -16,7 +16,7 @@
  *
  */
 
-// Binary client is an example client.
+// Binary client demonstrates how to send and receive metadata to and from RPC.
```

Comment:
Nit: Follow a consistent style? `read` and `set` are used on the server, while `send` and `receive` is used here. And `RPC` is used here, while `RPCs` are used on the server.

Reply:
Done

Reply:
Nit: `an RPC`?

Reply:
Done